python/samples/concepts/plugins/openai_function_calling_with_custom_plugin.py

```diff
@@ -104,7 +100,7 @@ async def main():
     settings: OpenAIChatPromptExecutionSettings = kernel.get_prompt_execution_settings_from_service_id(
         service_id=service_id
     )
-    settings.function_call_behavior = FunctionCallBehavior.EnableFunctions(
+    settings.function_choice_behavior = FunctionChoiceBehavior.Auto(
         auto_invoke=False, filters={"included_plugins": ["weather", "time"]}
```

Comment:
In this sample, this particular line is to showcase how one can handle manual tool calls. The `Auto` FunctionChoiceBehavior is to be understood that the model has functions advertised and it can pick to invoke or not invoke any of them to satisfy the user's query/goal.